packages/flutter/lib/src/material/app.dart

```diff
@@ -986,6 +986,16 @@ class _MaterialAppState extends State<MaterialApp> {
         },
       );
     }
+
+    childWidget = ScaffoldMessenger(
+      key: widget.scaffoldMessengerKey,
+      child: DefaultSelectionStyle(
+        selectionColor: effectiveSelectionColor,
+        cursorColor: effectiveCursorColor,
+        child: childWidget,
+      ),
```

Comment:
Probably not in the scope of this PR, but I noticed that the `Theme` widget already builds a `DefaultSelectionStyle`:

https://github.com/flutter/flutter/blob/16622e67a051a46dc0636df0e06322644411b92d/packages/flutter/lib/src/material/theme.dart#L132-L142

Would be nice to remove the duplicate widget here at some point :)